kbase-extension/static/kbase/js/common/jobs.js

```diff
@@ -169,11 +194,100 @@ define(['common/errorDisplay', 'common/format', 'common/html', 'common/ui'], (
      * @returns {boolean} true|false
      */
     function isValidJobInfoObject(jobInfo) {
+        const required = ['app_id', 'app_name', 'batch_id', 'job_id', 'job_params'];
+        try {
+            if (
+                jobInfo !== null &&
+                typeof jobInfo === 'object' &&
```

Comment:
Could just use your shiny new Util function here to test jobInfo type!

Reply:
doh, of course!

Reply:
fixed in https://github.com/kbase/narrative/pull/2646